tests/unit_test/apis/analytix_test.py

```diff
@@ -64,32 +64,35 @@
         TypeError,
         f"expect data_type to be an instance of AnalyticsDataType, but got {type('')}.",
     ),
-    (
-        "tag",
-        1.0,
-        AnalyticsDataType.SCALAR,
-        [1],
-        TypeError,
-        f"expect kwargs to be an instance of dict, but got {type(list())}.",
-    ),
+    # (
+    #     "tag",
+    #     1.0,
+    #     AnalyticsDataType.SCALAR,
+    #     [1],
+    #     TypeError,
+    #     f"expect kwargs to be an instance of dict, but got {type(list())}.",
+    # ),
 ]
```

Comment:
Why we comment this out?
if not applicable we should remove.
if applicable we should fix the test.